build/wd_test.bzl

```diff
@@ -63,6 +63,9 @@ def _wd_test_impl(ctx):
     # can't just specify some other executable with some args. OK, fine, we'll use a script that
     # just execs its args.
     if is_windows:
+        if len(ctx.attr.env):
```

Comment:
This will just make the Windows build fail right? Instead, you could use `target_compatible_with` to mark targets as incompatible with Windows if env is set

Reply:
Well, just passing env on non-Windows is probably easier and avoids just skipping the tests on Windows (assuming we won't have windows-asan in the future).